### DISCUSSION TITLE
Lailloken UI for PoE2

### DISCUSSION THREAD

**Discussion by EmmettJayhart** [Lailloken/Exile-UI#485] [Q&A] [answered]
Hey, I've absolutely fallen in love with Lailloken UI and I'm wondering if there are any plans to bring it to PoE2?
Maybe after early access ends? Or maybe you're not interested in a PoE2 version?

**Comment by Lailloken** [comment, marked as answer]
Glad to hear you like it. The tool actually already has PoE2 support "in theory" -- it's all just placeholder-ish because I didn't manage to get into one of the beta tests. You'll be able to use the same tool, and it'll use two separate sets of config-files and enabled features, depending on which game you're currently playing.

I always skip launch-evenings because I don't want to ruin my sleep schedule, so I spend that time tweaking stuff that may be affected by the new content introduced in the launch. For EA, I'll go through these placeholder implementations and confirm they're working, then see what else can be carried over from the PoE1 feature set.

If I'm lucky, I'll be able to release an initial compatibility update on Saturday morning (CET) which includes client-agnostic features:
- clone-frames
- notepad, timers
- search-strings
- cheat-sheet toolkit

Item-tooltips also look very similar, so I might be able to squeeze the context-menu in for quick-access to the new wiki -- assuming the game still lets us copy item-texts into clipboard.
